File: i2f-secure/web/secure-web/src/secure/data/secure-header.js
/**
 * 请求头sswh的解码组成部分
 */
const SecureHeader={
    // 签名
    sign: null,
    // 一次性消息
    nonce: null,
    // 随机秘钥
    randomKey: null,
    // Asym公钥签名
    asymSign: null,
    newObj: () => {
        return {
            sign: null,
            nonce: null,
            randomKey: null,
            asymSign: null
        }
    }
}
export default SecureHeader

File: i2f-secure/src/main/java/i2f/springboot/secure/data/SecureHeader.java
package i2f.springboot.secure.data;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author Ice2Faith
 * @date 2023/6/13 10:11
 * @desc
 */
@Data
@NoArgsConstructor
public class SecureHeader {
    public String sign;
    public String nonce;
    public String randomKey;
    public String asymSign;
}
